packages/docusaurus-theme-classic/src/theme/Layout/styles.css

```diff
@@ -10,11 +10,6 @@ body {
   height: 100%;
 }
 
-body {
-  margin: 0;
```

Comment:
No needed, since Infima already resets value for body margin https://github.com/facebookincubator/infima/blob/488499b987ad71e00285f5b0433f871b6b3d09dc/packages/core/dist/css/default/default.css#L366